plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -695,7 +695,44 @@ function OPDSBrowser:downloadFile(item, filename, remote_url)
     end
 end
 
-function OPDSBrowser:streamPages(item, remote_url, count)
+-- This function shows a dialog with input fields
+-- for entering information for Page streaming from.
+-- an input page number
+function OPDSBrowser:jumpToPage(viewer, count)
+    local input_dialog
+    input_dialog = InputDialog:new{
+        title = "Enter Page Number",
```

Comment:
```suggestion
        title = _("Enter page number"),
```

The _() is for translation, the string itself so it meets our style guide and will simply take the same translation as the other however many times it occurs.

Reply:
I figured it was there for a reason, just didn't know exactly why. That's why I tried copying other code that was already in the opdsbrowser.lua file. I'll re-add it.

Reply:
You can just click commit suggestion here in GitHub and then pull locally.